### PR TITLE
revert: add usage examples to man page

### PR DESCRIPTION
**Pull request by Woi**
@fichtner As of https://github.com/opnsense/core/issues/1804
Please review carefully, as I haven't build a man page from it.

### CODE REVIEW

File: revert/opnsense-revert.8 [hunk 1]
```diff
@@ -49,10 +50,20 @@ Ignore the signature verification result.
 Select the
 .Ar release
 of the package to be installed.
-Note that the release is not the individual package version.
+Note that the release is not the individual package version,
+but the version found in said OPNsense release.
 .El
 .Sh EXIT STATUS
 .Ex -std
+.Sh USAGE EXAMPLES
```
Comment (fichtner) [inline revert/opnsense-revert.8]: the preferred section name is "EXAMPLES"

File: revert/opnsense-revert.8 [hunk 2]
```diff
@@ -1,5 +1,6 @@
 .\"
 .\" Copyright (c) 2016-2017 Franco Fichtner <franco@opnsense.org>
+.\" Copyright (c)      2017 Marco Woitschitzky <woi@posteo.de>
```
Comment (fichtner) [inline revert/opnsense-revert.8]: No Alignment here, just a single whitespace :)

File: revert/opnsense-revert.8 [hunk 3]
```diff
@@ -49,10 +50,20 @@ Ignore the signature verification result.
 Select the
 .Ar release
 of the package to be installed.
-Note that the release is not the individual package version.
+Note that the release is not the individual package version,
+but the version found in said OPNsense release.
```
Comment (fichtner) [inline revert/opnsense-revert.8]: "but rather"

File: revert/opnsense-revert.8 [hunk 4]
```diff
@@ -49,10 +50,20 @@ Ignore the signature verification result.
...
 .El
 .Sh EXIT STATUS
 .Ex -std
+.Sh USAGE EXAMPLES
+Revert unbound to the version found in OPNsense 17.7.1:
+opnsense-revert -r 17.7.1 unbound
```
Comment (fichtner) [inline revert/opnsense-revert.8]: Use this for shell blocks according to OpenBSD:

.Pp
.Dl # opnsense-revert -r 17.7.1 unbound
.Pp

File: revert/opnsense-revert.8 [hunk 5]
```diff
@@ -49,10 +50,20 @@ Ignore the signature verification result.
...
+.Pp
+Revert OPNsense inculding all packages back to release 17.7:
```
Comment (fichtner) [inline revert/opnsense-revert.8]: That's not true, "opnsense" is the GUI/configuration package ;)

File: revert/opnsense-revert.8 [hunk 6]
```diff
@@ -49,10 +50,20 @@ Ignore the signature verification result.
...
+opnsense-revert -r 17.7 opnsense
+.Pp
+Bring back unbound to the current version:
```
Comment (fichtner) [inline revert/opnsense-revert.8]: "Bring the package back"

Best to stick to one example package to avoid confusion.

**Comment by Woi** [comment]
Hi Franco,

thanks for the review. I incorporated most of the requested changes. However I'd like to keep an example on how to revert everything to a given OPNsense release or make clear that this is not possible. Is there something like `opnsense-revert -r 17.7 *`?

File: revert/opnsense-revert.8 [hunk 7]
```diff
@@ -49,10 +50,24 @@ Ignore the signature verification result.
...
-Note that the release is not the individual package version.
+Note that the release is not the individual package version,
+but rather the version found in said OPNsense release.
 .El
 .Sh EXIT STATUS
 .Ex -std
+.Sh EXAMPLES
+Revert unbound to the version found in OPNsense 17.7.1:
+.Pp
+opnsense-revert -r 17.7.1 unbound
```
Comment (fichtner) [inline revert/opnsense-revert.8]: Missing:

.Dl #

File: revert/opnsense-revert.8 [hunk 8]
```diff
@@ -49,10 +50,24 @@ Ignore the signature verification result.
...
+.Pp
+Bring unbound back to the current version:
+.Pp
+.Dl # opnsense-revert unbound
+.Pp
+.Dl # Revert OPNsense inculding all packages back to release 17.7:
```
Comment (fichtner) [inline revert/opnsense-revert.8]: Too much:

.Dl #

**Comment by fichtner** [comment]
I forgot to say: Hi Marco! :)

**Comment by fichtner** [comment]
Merged, thanks!

**Comment by Woi** [comment]
Thanks for the explanation . Should be fine now... ah it is :)
You're welcome and thanks for the merge :-)

BTW: I usually prefer Woi over Marco.

**Comment by fichtner** [comment]
Alright, woi it is.

Minor tweak for style / clarity was committed as well: https://github.com/opnsense/update/commit/c6c44bce6a

**Pull request merged**